lib/active_admin/resource/action_items.rb

```diff
@@ -110,6 +110,10 @@ def initialize(name, options = {}, &block)
     def html_class
       "action_item #{@options[:class]}".rstrip
     end
+
+    def priority
+      @options[:priority] || 10
```

Comment:
@andreslemik  please fix docs for add_action_item method with including description for new option

Reply:
@Fivell, all done